extensions/industry/healthcare/fhir/datatypes/address.schema.json

```diff
@@ -0,0 +1,100 @@
+{
+   "meta:license":[
+      "Copyright 2024 Adobe Systems Incorporated. All rights reserved.",
+      "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+      "you may not use this file except in compliance with the License. You may obtain a copy",
+      "of the License at https://creativecommons.org/licenses/by/4.0/"
+   ],
+   "$schema":"http://json-schema.org/draft-06/schema#",
+   "$id":"https://ns.adobe.com/xdm/industry/healthcare/fhir/datatypes/address",
+   "title":"Address",
+   "type":"object",
+   "description":"An address expressed using postal conventions (as opposed to GPS or other location definition formats).(This datatype is created as per HL7 FHIR Release 5 specifications)",
+   "definitions":{
+      "address":{
+         "properties":{
+            "xdm:use":{
+               "title":"Use",
+               "description":"home | work | temp | old | billing - purpose of this address",
+               "type":"string",
+               "enum":[
+                  "home",
+                  "work",
+                  "temp",
+                  "old",
+                  "billing"
+               ],
+               "meta:enum":{
+                  "home":"Home",
+                  "work":"Work",
+                  "temp":"Temp",
+                  "old":"Old",
+                  "billing":"Billing"
+               }
+            },
+            "xdm:type":{
+               "title":"Type",
+               "description":"postal | physical | both",
+               "type":"string",
+               "enum":[
+                  "postal",
+                  "physical",
+                  "both"
+               ],
+               "meta:enum":{
+                  "postal":"postalCode",
```

Comment:
Minor comment: You could change it to "Postal Code" since this text is displayed in the UI